### ISSUE TITLE
Consider moving away from Ansible

### ISSUE THREAD

**Issue by wincent** [wincent/wincent#82]
I like it much better than any system configuration tool I've used in the past (that is, Puppet and then Chef), but I am [scared of the complexity of the dependency tree](https://twitter.com/wincent/status/1239980516755079170). I get nervous every time I have to update, and that sucks. My entire programming career can be summed up as a war against the forces of entropy, and I don't exactly want *my frickin' dotfiles repo* to be the site of a losing battle in that fight.

Given that I am using it to do simple setup of one (local!) machine, I think it may be overkill, kind of like using a 10,000-pound hammer to drive in a nail. This issue is just a wishlist item for now, but it is something to think about.

**Comment by wincent** [comment]
Going to close this. Not done yet, but we're well past the "considering" phase at this point.

**Comment by nomasprime** [comment]
Hey @wincent 👋 

Discovered your work a while back through your YouTube channel, really good stuff. Came here and you were the first other person I've found using Ansible for their dotfiles.

I started out with Puppet about seven years ago, then Ansible, Salt, and back to Ansible because it works better for configuring localhost.

Now I see you've gone one step further and written your own provisioner, Fig—wow.

I understand your motivation but wondering how you feel about it now: is addressing these issues worth the extra development and maintenance time; other than stated in the Readme, have you discovered any other advantages; any disadvantages?
